src/include/misc.h

```diff
@@ -44,6 +44,12 @@
 
 #define WT_MINUTE (60)
 
+/* Time (in minutes) the cache is stuck before WT crashes in diagnostic mode.*/
+#define WT_CACHE_STUCK_TIMEOUT (5)
```

Comment:
If you define these exactly what you want, you won't need variables to calculate when using:

#define WT_CACHE_STUCK_TIMEOUT (300)
/* Enable verbose messages 2 seconds before calling cache as stuck. */
#define WT_CACHE_STUCK_ENABLE_VERBOSE (WT_CACHE_STUCK_TIMEOUT - 2)

Now in the code, you can directly compare against these and won't need the variable `cache_stuck_timeout`.